gcp-bundle/NOTICE

```diff
@@ -241,3 +241,111 @@ NOTICE for Group: org.conscrypt  Name: conscrypt-openjdk-uber  Version: 2.5.2
 |     * licenses/LICENSE.harmony.txt (Apache License 2.0)
 |   * HOMEPAGE:
 |     * https://harmony.apache.org/
+--------------------------------------------------------------------------------
+
+NOTICE for Group: org.apache.httpcomponents  Name: httpclient  Version: 4.5.14
+
+| Notice: Apache HttpClient
```

Comment:
Just FYI, it's good here, but it's not required to copy all ASF dependencies NOTICE file, because, the first section of the NOTICE contains `This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).`